pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/SingularFileUploadNotifications.java

```diff
@@ -0,0 +1,106 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dao.MailTemplateDao;
+import org.broadinstitute.ddp.db.dao.MailTemplateRepeatableElementDao;
+import org.broadinstitute.ddp.db.dto.MailTemplateDto;
+import org.broadinstitute.ddp.db.dto.MailTemplateRepeatableElementDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.sqlobject.SqlObject;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Task to update existing singular study by specifying newly added mail template
+ */
```

Comment:
Let's update this comment to mention the fact that if a pre-existing template is set for the study, it will be deleted.